app/controllers/people_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    render :person
```

Comment:
The Rails convention is to use a view with the same name as the action. So if you changed the `person.html.erb` template to be `show.html.erb` the you would not even need to call `render` here — Rails would do that for you.

Incidentally, you don't need the `render` on line 5 because your view template has the same name as the action.